File: app/src/main/java/zelpage/razeni/bo/Request.java
package zelpage.razeni.bo;

import org.json.JSONException;
import org.json.JSONObject;

/**
 * BO for razeni query
 * @author jindrichbr
 */
public class Request {

    private String trainNumberQuery;

    public String getTrainNumberQuery() {
        return trainNumberQuery;
    }

    public void setTrainNumberQuery(String trainNumberQuery) {
        this.trainNumberQuery = trainNumberQuery;
    }

    public JSONObject ToJson() throws JSONException {
        JSONObject result = new JSONObject();
        result.put("TrainNumberQuery", this.trainNumberQuery);

        return result;
    }
}

File: app/src/main/kotlin/zelpage/razeni/bo/Request.kt
package zelpage.razeni.bo

import org.json.JSONException
import org.json.JSONObject

/**
 * BO for razeni query
 */
data class Request(
	var trainNumberQuery: String? = null) {

	@Throws(JSONException::class)
	fun ToJson(): JSONObject {
		val result = JSONObject()
		result.put("TrainNumberQuery", this.trainNumberQuery)

		return result
	}

}
